agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/IntegrationTests.java

```diff
@@ -114,10 +113,10 @@ public void integrationTest() throws Exception {
     }
 
     ExecutorService executorService = Executors.newFixedThreadPool(10);
-    for (int i = 0; i < 10; i++) {
+    for (int i = 0; i < 1; i++) {
```

Comment:
for loop is not needed for 1 iteration.

Reply:
oops! this was just a local change for testing, reverted, thx!